arches_component_lab/views/api/widgets.py

```diff
@@ -69,8 +67,17 @@ def get(self, request, graph_slug, node_alias):
 
 class NodeDataView(View):
     def get(self, request, graph_slug, node_alias):
-        node = models.Node.objects.get(
-            graph__slug=graph_slug, alias=node_alias, source_identifier_id__isnull=True
+        node = (
```

Comment:
We can probably avoid a hard version check here by doing something like getattr(Node, 'source_identifier_id')